pkg/network/protocols/http2/model_linux.go

```diff
@@ -88,8 +88,15 @@ func decodeHTTP2Path(buf [maxHTTP2Path]byte, pathSize uint8) ([]byte, error) {
 	return []byte(str), nil
 }
 
+// ebpfTXWrapper is a wrapper around the eBPF transaction.
+// It extends the basic type with a pointer to an interned string, which will be filled by processHTTP2 method.
+type ebpfTXWrapper struct {
+	*EbpfTx
+	dynamicTable *DynamicTable
+}
```

Comment:
General note which I do not expect to be part of this PR but just to rise: Isn't this object and wrapper may have a major influence in our current process? this struct would be really big with a lot of content